prisma/schema.prisma

```diff
@@ -203,4 +203,4 @@ model ApiUser {
 
   @@map("api_users")
   @@index([api_key])
-}
+}
```

Comment:
just check out the unchanged version of the file or add a newline please, this file shouldn't show up in the diff

Reply:
bump